charts/node-analyzer/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: node-analyzer
 description: Sysdig Node Analyzer
 
 # currently matching Sysdig's appVersion 1.14.34
-version: 1.12.0
-appVersion: 12.6.0
+version: 1.12.1
```

Comment:
I suppose should be a minor since we are bumping the app from 1.26.0 to 1.27.0, right?

Reply:
@mavimo you're right. I'll change to a minor release